src/Blackguard/Blackguard.csproj

```diff
@@ -38,6 +42,23 @@
     <ResourceFolder>Resources/Linux</ResourceFolder>
   </PropertyGroup>
 
+  <!-- Check make is available before trying to do anything -->
+  <Target Name="CheckMakeBinaryCallable">
+    <Exec Command="$(MakeBinary) -h > NUL" Condition="$(IsWindows)" IgnoreExitCode="True">
+      <Output TaskParameter="ExitCode" PropertyName="MakeExitCode" />
+    </Exec>
+    <Exec Command="$(MakeBinary) --help > /dev/null" Condition="$(IsLinux)" IgnoreExitCode="True">
+      <Output TaskParameter="ExitCode" PropertyName="MakeExitCode" />
+    </Exec>
+  </Target>
+
+  <!-- Compile PDCurses if targeting windows -->
+  <Target Name="Compile PDCurses" BeforeTargets="BeforeBuild" Condition="$(BuildingWindows)" DependsOnTargets="CheckMakeBinaryCallable">
```

Comment:
This should run even if an os-flag is not set when compiling (such as building normally on windows).

Reply:
Addressed in [b41992d](https://github.com/ppebb/isp2/pull/80/commits/b41992d2be11f1deb504803ea0f4d27e12cba77e)